resources/views/pages/ticket/[ticket].blade.php

```diff
@@ -317,37 +317,39 @@ function AwardManually(hardcore) {
         </div>
 
         <div class="commentscomponent">
-            <table id="feed" class="table-highlight"><tbody>
-                <x-comment.item
-                    :author="$ticket->reporter"
-                    :when="$ticket->ReportedAt"
-                    :payload="$ticketNotes"
-                />
-                @php $numArticleComments = getRecentArticleComments(ArticleType::AchievementTicket, $ticket->ID, $commentData) @endphp
-                @php $allowDelete = $permissions >= Permissions::Moderator @endphp
-                @foreach ($commentData as $comment)
-                    @php
-                        $when = Carbon::createFromTimestamp($comment['Submitted']);
-                        $commentUser = 
-                            ($comment['User'] === $ticket->reporter->User) ? $ticket->reporter :
-                            (($comment['User'] === $user->User) ? $user :
-                                User::firstWhere('User', $comment['User']));
-                    @endphp
+            <table id="feed" class="table-highlight">
+                <tbody>
```

Comment:
This is all just fixed indention. `<table>` and `<tbody>` were indented the same, so I moved this `<tbody>` to the next line.